lib/config.js

```diff
@@ -42,6 +42,8 @@ config.externalGradingHistoryLoadIntervalSec = 3600;
 config.externalGradingCurrentCapacityFactor = 1.5;
 config.externalGradingHistoryCapacityFactor = 1.5;
 config.externalGradingSecondsPerSubmissionPerUser = 120;
+config.fileEditorUseAws = false;
+config.fileEditorS3Bucket = 'prairielearn.dev.file-edits';
```

Comment:
I've used `.prod` and `.staging` now instead of `.` and `.dev`. With respect to case, the convention I'm using is identical to what has been used - any future changes are welcome, and can be applied later.